### ISSUE TITLE
Number of nodes and shape of node coordinates differ by 1

### ISSUE THREAD

**Issue by diamondspark** [a-r-j/graphein#98]
Hi @a-r-j 

I've been observing that some times the number of nodes generated by the library differs to the coordinate data generated by it by exactly 1 node. Do you know why this happens. Following is an example code

```
configs = {
        "granularity": "CA",
        "keep_hets": False,
        "insertions": False,
        "verbose": False,
        "dssp_config": DSSPConfig(),
        "node_metadata_functions": [meiler_embedding,expasy_protein_scale],
        "edge_construction_functions": [add_peptide_bonds,
                                                  add_hydrogen_bond_interactions,
                                                  add_ionic_interactions,
                                                  add_aromatic_sulphur_interactions,
                                                  add_hydrophobic_interactions,
                                                  add_cation_pi_interactions]
        }
config = ProteinGraphConfig(**configs)
format_convertor = GraphFormatConvertor('nx', 'pyg', 
                                            verbose = 'gnn', 
                                            columns = None)
g = construct_graph(config=config, pdb_code='1c5y')
protdata = format_convertor(g)
print(protdata)
```
protdata.num_nodes == 256 ; protdata.coords[0].shape==257

Shouldn't these 2 be the same? What am I missing?
Thank you!

**Comment by a-r-j** [comment]
Yep, you're right @diamondspark . There was a bug in how inserted residues are removed from the dataframe. I've pushed a fix to a pending PR. Need to write some tests but hope to get this merged in soon!

There should actually be 238 nodes in the graph as we remove the inserted residues.

**Comment by diamondspark** [comment]
Thank you for looking into this. How to get the updated code. I did pip install graphein --upgrade. That didn't change the code in protein/graphs.py

**Comment by a-r-j** [comment]
Hi, @diamondspark. I'm working on this. It's a tricky problem resulting from insertions and alt_locs in the PDB files. These aren't always consinstently represented in the file so it's hard to come up with a robust way that catches all the corner cases.

While I'm figuring it out you can try pre-processing the PDBs with the excellent [PDB Tools](http://www.bonvinlab.org/pdb-tools/).

**Comment by a-r-j** [comment]
Hi @diamondspark I believe this is resolved in `1.1.1` - try it out & do let me know if not!
